src/features/home/api/topFiveStacks.api.ts

```diff
@@ -0,0 +1,14 @@
+import { StackData } from '@/features/home/model/topFiveStacks.model';
+
+export const fetchData = async (): Promise<StackData[] | undefined> => {
+  try {
+    const response: Response = await fetch('/data/top5Stacks.json', {
```

Comment:
혹시 요렇게 표현해줘야하지 않을까요?
```suggestion
    const response: Response = await fetch('/data/topFiveStacks.json', {
```

Reply:
오타였네요! 감사합니다!! https://github.com/GrebbMe/grabbme-fe/pull/87/commits/c5cf421bbbedc1109d2527010c816f9b38c15742